### ISSUE TITLE
If we close the Notification[Toastr] on one tab, can we get notified on another tab and the toastr gets dismissed there as well.

### ISSUE THREAD

**Issue by aadilwani** [CodeSeven/toastr#540]
If we close the Notification[Toastr] on one tab, can we get notified on another tab and the toastr gets dismissed there as well. ?

**Comment by mohanen** [comment]
No, you have to write your logic to do that in the toastr.options.onclick call back.
You can use cookies or session storage or db to handle the state of the notifications.
